cvmfs/server/cvmfs_server_migrate.sh

```diff
@@ -479,6 +479,21 @@ _migrate_141() {
   load_repo_config $name
 }
 
+_migrate_142() {
```

Comment:
Actually, now that template transactions are merged the `_migrate_142` from there can be reused.